### ISSUE TITLE
LP filter accelerometer data help

### ISSUE THREAD

**Issue by arcgislearner** [xioTechnologies/Gait-Tracking-With-x-IMU#27]
Hello, I want to know how to determine the value of the filtCutOff in Script.m：

filtCutOff =5;
[b, a] = butter(1, (2*filtCutOff)/(1/samplePeriod), 'low');
acc_magFilt = filtfilt(b, a, acc_magFilt);

**Comment by MTfirst** [comment]
In my experience, about 1 Hz to 6 Hz is brought by subjects.
In other words, f < about 1 or f > about 6 is noise (for example: offset of IMU) so we can cut.

Though filtCutOff=5 is valid of most subjects, if you want define manually, Upper Band should determined according to walking speed.

Thanks.

**Comment by arcgislearner** [comment]
OK,thanks for your reply, i will try it again.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"MTfirst"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月10日(星期二) 凌晨4:39
收件人:&nbsp;"xioTechnologies/Gait-Tracking-With-x-IMU"<Gait-Tracking-With-x-IMU@noreply.github.com&gt;;
抄送:&nbsp;"philrain"<cjh19970707@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xioTechnologies/Gait-Tracking-With-x-IMU] LP filter accelerometer data help (#27)




In my experience, about 1 Hz to 6 Hz is brought by subjects.
 In other words, f < about 1 or f &gt; about 6 is noise (for example: offset of IMU) so we can cut.
 
Though filtCutOff=5 is valid of most subjects, if you want define manually, Upper Band should determined according to walking speed.
 
Thanks.

**Comment by Abdul-Sajeed** [comment]
Hello, what if i am using vehicle with speed of 20 - 30 km/hr. Is it possible to reset cutoff speed ? (thanks)

**Comment by arcgislearner** [comment]
Sorry, I didn't do this in the car, I just tied it to my feet.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xioTechnologies/Gait-Tracking-With-x-IMU"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月10日(星期六) 凌晨1:36
收件人:&nbsp;"xioTechnologies/Gait-Tracking-With-x-IMU"<Gait-Tracking-With-x-IMU@noreply.github.com&gt;;
抄送:&nbsp;"Philrain"<cjh19970707@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xioTechnologies/Gait-Tracking-With-x-IMU] LP filter accelerometer data help (#27)





 
Hello, what if i am using vehicle with speed of 20 - 30 km/hr. Is it possible to reset cutoff speed ? (thanks)
